sintr_analysis_server_example/test_driver.dart

```diff
@@ -85,3 +69,55 @@ main(List<String> args) async {
   var reduced = completionReductionMerge(reduced1, reduced2);
   print(reduced);
 }
+
+Future<Map> loadSessionInfo(String path) async {
```

Comment:
I'd rather we processed this from a text string rather than a file. It's not always going to be available via IO -- e.g. if it comes from cloud storage, we'd probably prefer to read it as an in-memory string.